pallets/manta-pay/src/lib.rs

```diff
@@ -600,12 +634,36 @@ pub mod pallet {
             Self::deposit_event(
                 config::TransferPost::try_from(post)
                     .map_err(|_| Error::<T>::InvalidSerializedForm)?
-                    .post(sources, sinks, &(), &mut Ledger(PhantomData))
+                    .post(
+                        &load_transfer_parameters(),
+                        &mut Ledger(PhantomData),
+                        &(),
+                        sources,
+                        sinks,
+                    )
                     .map_err(Error::<T>::from)?
                     .convert(origin),
             );
             Ok(().into())
         }
+
+        ///
+        #[inline]
+        pub fn id_from_field(id: [u8; 32]) -> Option<StandardAssetId> {
+            if u128::from_le_bytes(Array::from_iter(id[16..32].iter().copied()).into()) == 0 {
```

Comment:
Not sure why not use `id[16..32]` directly.

Reply:
How about `0u128.to_le_bytes() == id[16..32]`?
`to_le_bytes` is a const method.

Reply:
yeah that looks nicer